frontend/app/services/tour.js

```diff
@@ -159,8 +159,8 @@ export default class TourService extends Tour {
               await this.router.transitionTo(
                 this.autostartTour.undoneTours.shift() ?? "index"
               );
-            } catch (error) {
-              this.notify.error("unexpected error");
+            } catch (_) {
+              _.toString();
             }
```

Comment:
this:

1. doesn't fix the underlying issue
2. the `_.toString()` doesn't do anything and is not necessary

Since this is important for onboarding tho, we can probably use this as a **temporary** solution
```suggestion
            } catch {}
```

(i haven't tested the suggestion but it should be equivalent to the proposed **temporary** solution

Reply:
i haven't done frontend in a while, you can probably just add a **temporary** eslint ignore comment

Reply:
But yes, eslint is right, we shouldn't have empty block statements

Reply:
AFAIK, especially in try catch statements, an empty (catch) block is an antipattern

Reply:
@derrabauke what would you say?

Reply:
Why fix the symptom and not the root cause? @MitanOmar, why is the transition aborted in the first place?

Reply:
@derrabauke i really don't know, but when i put `this.router.transitionTo` inside any try catch, even not there, it's throwing this error

Reply:
@derrabauke for example, in the home page, i change the routing from
```hbs
          <LinkTo @route="index.attendances">
            Attendances
            <span class="badge">{{humanize-duration this.attendanceSum}}</span>
          </LinkTo>
```
to be an action in the controller with `this.router.transitionTo` with try catch, it was throwing the same error again

Reply:
I've checked it. Just remove the `try-catch` block. It's not necessary at this point.

Reply:
@derrabauke done

Reply:
Sure you pushed the changes? O.o

Reply:
yes, this is the final results :)

Reply:
![image](https://github.com/adfinis/timed/assets/10029904/aaf91b66-680c-42b4-bb11-c2043bc67c14)

the catch block has still some non-sense code in it. Would expect `catch {}` with some eslint exception comment in it!?

Reply:
@derrabauke  it's empty now